megamek/src/megamek/client/ui/swing/lobby/ChatLounge.java

```diff
@@ -2835,9 +2837,9 @@ public class MekTableMouseAdapter extends MouseInputAdapter {
         public void mouseClicked(MouseEvent e) {
             if (e.getClickCount() == 2) {
                 int row = mekTable.rowAtPoint(e.getPoint());
-                Entity entity = mekModel.getEntityAt(row);
-                if (entity != null && isEditable(entity)) {
-                    lobbyActions.customizeMech(entity);
+                InGameObject entity = mekModel.getEntityAt(row);
+                if (entity != null && entity instanceof Entity && isEditable((Entity) entity)) {
```

Comment:
```suggestion
                if ((entity instanceof Entity) && isEditable((Entity) entity)) {
```
`instanceof` acts as a null check